blog/2015-02-26-Getting-Started-With_Docker-And-Wildfly.adoc

```diff
@@ -0,0 +1,195 @@
+= Getting started with Docker and Wildfly
+:page-layout: blog
+:page-author: rstryker
+:page-tags: [jbosstools, devstudio, server, docker, wildfly]
+
+So you've heard the buzzwords and all the hype around link:http://www.docker.com[Docker],
+but you haven't had the chance to play around with it yet or see if it fits your needs. 
+You're already an avid user of JBossTools, and are fairly familiar with starting and stopping
+your local or remote Wildfly installation, and deploying web applications to it. 
+
+If this describes you, and you're interested in trying out Docker, then this blog is targeted to you. 
+
+== Docker Installation
+
+Docker's installation method differs based on your platform. You'll want to 
+review link:https://docs.docker.com/installation/#installation[Docker's Installation] guides
+for your respective platforms. 
+
+== Running the default wildfly image
+
+Wildfly has published their docker images for public consumption. These images are very standard, 
```

Comment:
add link to https://registry.hub.docker.com/u/jboss/wildfly/ and jboss.org/docker ?